web_src/css/shared/settingtable.css

```diff
@@ -0,0 +1,39 @@
+/* This css is for table elements inside admin, organization, and repository settings 
+  TODO: Right now there is no table inside user settings, but in the future if there is table
+  in user settings, should also use this common style here.
+*/
+.admin .table.segment,
+.repository.settings .table.segment,
+.organization.settings .table.segment {
```

Comment:
I think we can introduce a general `<div class="twelve wide column g-grid-setting-right">` for every "layout_head", then we only need to write styles for `g-grid-setting-right`

Reply:
Good idea, I will add this class